view/template/report/dmca.php

```diff
@@ -1,3 +1,4 @@
+<?php header("Access-Control-Allow-Origin: *"); ?>
```

Comment:
I'm going to be opaque in my response for teaching a man to fish purposes.

How are headers set elsewhere in the site? If you don't know, how would you go about answering that question?

Reply:
Response::setHeader("Acces-Control-Allow-Origin", "*");